extra/migrations/20160411000000_migrate_libvirt_settings.rb

```diff
@@ -0,0 +1,29 @@
+require 'fileutils'
+require 'yaml'
+
+class MigrateVirshToLibvirtConfig < ::Proxy::Migration
+  def migrate
+    input_yaml = YAML.load_file(File.join(src_dir, 'settings.yml'))
+    copy_original_configuration_except(path("dhcp_virsh.yaml", "dns_virsh.yml"))
```

Comment:
Oh good catch.

I still plan to remove virsh provider in another commit once this is done.